src/pymedphys/dcm/_level1/dcmdose.py

```diff
@@ -735,17 +658,3 @@ def contour_to_points(contours):
     contour_points = np.concatenate(resampled_contours, axis=1)
 
     return contour_points
-
-
-def check_dcmdose(dcm):
```

Comment:
Unneccessary inclusion. `pydicom` already includes checks like this. A simple check for modality is fine, i think.